src/components/TutorDashboard/index.js

```diff
@@ -405,24 +407,29 @@ function Tutor(props) {
         <Grid container className={classes.filters} mb={2}>
           <Grid item>
             <Button
-              variant="contained"
+              // "contained"
               className={classes.python}
               onClick={() => {
                 setVolunteer(filterPathway(1, cacheVolunteer));
                 setSelectedPathway("Python");
+                setPathwayClicked(true);
+                setPathwayTwoClicked(false);
```

Comment:
Now after making the above suggestion, you only need to set pathway clicked, the other one will automatically update.
```suggestion
```